R/add_phylopic_tree.R

```diff
@@ -0,0 +1,84 @@
+#' Add PhyloPics to a plotted phylogenetic tree
+#'
+#' Specify existing images, taxonomic names, or PhyloPic uuids to add PhyloPic
+#' silhouettes alongside the associated leaves of a phylogenetic tree.
+#'
+#' @inheritParams add_phylopic
+#' @param tree The phylogenetic tree object of class `phylo` on which to add
+#' the silhouette.
+#' @param tip The tip labels against which to add the silhouettes.
+#' @param relWidth The width of each silhouette relative to the plotting area.
+#' @param padding Distance to inset each silhouette from the right edge of the
+#' plotting area, relative to the size of the plotting area.
+#' Negative values offset to the right.
```

Comment:
I believe all of the arguments in rphylopic use coordinate space, not relative space, so it would be good to be consistent with that.

Reply:
Per `relWidth`, coordinate space is less useful in a tree where it is outwith the user's control.  Perhaps offering the user `padding` and `relPadding` would be the way to go?